cmd/root.go

```diff
@@ -99,6 +105,31 @@ func createApp(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	if !skipCheck {
```

Comment:
We should only check this like once a day

Reply:
Provided a proposal, please review